test/e2e/fixture-builder.js

```diff
@@ -594,10 +591,7 @@ class FixtureBuilder {
               caveats: [
                 {
                   type: 'restrictReturnedAccounts',
-                  value: [
-                    '0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
```

Comment:
This is the default selected account. I can update this in a separate PR to use the variable here instead.